src/components/Top-Bar.vue

```diff
@@ -1,82 +1,73 @@
 <template>
-  <oc-navbar id="oc-topbar" tag="header" class="oc-topbar">
-    <oc-navbar-item position="left">
-      <oc-button icon="menu" variation="primary" class="oc-topbar-menu-burger uk-height-1-1" aria-label="Menu" @click="toggleSidebar(!isSidebarVisible)" v-if="!publicPage()" ref="menubutton">
-        <span class="oc-topbar-menu-burger-label" v-translate>Menu</span>
-      </oc-button>
-    </oc-navbar-item>
-    <oc-navbar-item position="center">
-      <router-link to="/" class="oc-topbar-icon">ownCloud X</router-link>
-    </oc-navbar-item>
-    <oc-navbar-item position="right" v-if="!publicPage()">
-      <notifications v-if="activeNotifications"></notifications>
-      <div class="oc-topbar-personal">
-        <avatar class="oc-topbar-personal-avatar" :userid="user.id" />
-        <span class="oc-topbar-personal-label">{{ user.displayname }}</span>
-      </div>
-    </oc-navbar-item>
-  </oc-navbar>
+  <div>
```

Comment:
Is the wrapping div here needed?

Reply:
Removing. It's a remain from a previous attempt.